cloud_testing/kotlin_poc/src/test/kotlin/ftl/android/AndroidCatalogTest.kt

```diff
@@ -0,0 +1,24 @@
+package ftl.android
+
+import junit.framework.Assert.assertEquals
+import org.hamcrest.CoreMatchers.instanceOf
+import org.junit.Assert.assertThat
+import org.junit.Assert.assertTrue
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.ExpectedException
+
+class AndroidCatalogTest {
+
+    private val bitrise = System.getenv("BITRISE_IO") != null
+
+    @Test
+    fun validateDeviceConfig() {
+        if (bitrise) return
```

Comment:
The Android catalog isn't API restricted so this test should work on Bitrise.

Reply:
cool, I'll get rid of it then!

Reply:
...it's back!